pkgs/development/compilers/graalvm/community-edition/mkGraal.nix

```diff
@@ -1,7 +1,9 @@
-{ javaVersion
+{ config
 , defaultVersion
+, javaVersion
 , platforms
-, config
+, forceUpdate ? false
+, sourcesPath ? ./graalvm${javaVersion}-ce-sources.json
```

Comment:
This works locally but not on ofborg, so I can only assume it is because this is something introduced in the newer versions of nix and ofborg is using an older one. Or vice-versa.

Going back to the older (but uglier) concatenation of path + string.